neuralprophet/configure.py

```diff
@@ -398,12 +398,12 @@ def __post_init__(self):
             self.seasonality_local_reg = False
 
         # If seasonality_local_reg = True
-        if self.seasonality_local_reg:
+        if self.seasonality_local_reg is True:
             log.error("seasonality_local_reg = True. Default seasonality_local_reg value set to 1")
             self.seasonality_local_reg = 1
 
         # If Season modelling is global.
-        if self.global_local == "global" and self.seasonality_local_reg:
+        if self.global_local == "global" and self.seasonality_local_reg is True:
```

Comment:
Here,   your  change was actually correct before:
`        if self.global_local == "global" and self.seasonality_local_reg:`